app/Application.php

```diff
@@ -243,8 +247,9 @@ protected function authorize()
                     }
 
                     $user = $this->userRepository->create([
-                        'username' => $userName,
-                        'password' => $password
+                        'user_name' => $userName,
+                        'password' => $password,
+                        'role' => 'basic'
```

Comment:
let's move `basic` value to the constant inside the UserRepository class named `ROLE_BASIC`